### PR TITLE
Update gitignore

### PR DESCRIPTION
**Pull request by lukaszgryglicki**
@mlehotskylf - just wanted to make sure that some files I use locally are not committed to the repo.

### CODE REVIEW

**Review by mlehotskylf — DISMISSED** [review]

**Comment by lukaszgryglicki** [comment]
Still I can't merge @mlehotskylf due to CI failures not related to this PR - can u merge?

**Comment by lukaszgryglicki** [comment]
`lint-test-build` never finished, is this a bug @mlehotskylf - it is listed as "required" and didn't finish since yesterday...

**Comment by lukaszgryglicki** [comment]
@mlehotskylf this is still not resolved.

**Comment by lukaszgryglicki** [comment]
`lint-test-build` still blocks this @mlehotskylf

File: setenv.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+rm -rf /tmp/aws
+cp -R /root/.aws /tmp/.aws
+
+data="$(aws sts assume-role --role-arn arn:aws:iam::395594542180:role/product-contractors-role --profile lfproduct --role-session-name lfproduct-dev-session)"
```
Comment (nickmango) [inline setenv.sh:6]: @lukaszgryglicki you happen to use your iam arn role details here

Comment (nickmango) [reply setenv.sh:6]: Arent the scripts abit of work. We have collections that handle the endpoints. Weve moved away from postman but theres a recommendation to use bruno https://www.usebruno.com/ as a replacement

Comment (lukaszgryglicki) [reply setenv.sh:6]: This ARN is for "dev" when any of our users is assuming lfproduct-dev role - I can make that an env variable if needed.

Well I use `curl` locally, those scripts are not part of any CI or deployment, so I woudl like to have them in the repo, is this an issue? cc @mlehotskylf 

I can remove that ARN and make it a required ENV variable, but, again, it is not a secret, it is just the `dev product-contractors-role`...

Comment (lukaszgryglicki) [reply setenv.sh]: Anyway, changed to get `ARN` from the `.secret` file (`*.secret` files are gitignored) PTAL again, thanx.

**Review by mlehotskylf — DISMISSED** [review]

**Review by nickmango — APPROVED** [review]
lgtm

**Review by mlehotskylf — APPROVED** [review]

**Pull request merged**